index.html

```diff
@@ -18,16 +21,178 @@
             </div>
         </nav>
     </header>
-    <!-- END NAVBAR SECTION-->
+    <!-- End NAVBAR SECTION-->
 
-    <!-- START HEADLINE SECTION-->
+    <!-- Start HEADLINE SECTION-->
     <section class="headline">
         <div class="content">
             <h1 class="author">Thiri Htet Htet Aung</h1>
             <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
             <button type="button" class="collab-btn">Start collaboration</button>
         </div>
     </section>
-    <!-- END HEADLINE SECTION -->
+    <!-- End HEADLINE SECTION -->
+
+    <!-- Start WORK SECTION -->
+    <section class="works">
+        <!-- start title -->
+        <div class="title">
+            <h1>Projects</h1>
+        </div>
+        <!-- end title -->
+
+        <!-- start project card -->
+        <div class="projects">
+            <!-- start card1 -->
+            <div class="project-cards project1">
+                <div class="pjimgs">
+                    <img src="./img/pj1.svg" alt="pj1"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
+                </div>
+            </div>
```

Comment:
In order to have a flexible layout (which you would do in future milestones), it is required to align the contents inside each cards with flex.